plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -25,7 +24,7 @@ import java.util.concurrent.CopyOnWriteArrayList
  * to the user's current location.
  */
 class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
-  LocationComponentSettingsBase() {
+  LocationComponentAccuracyRadiusSettingsBase() {
```

Comment:
Why do we need that base class if we do not have `override` modifier for `addOnIndicatorAccuracyRadiusChangedListener` / `removeOnIndicatorAccuracyRadiusChangedListener`?

Reply:
The base class is related to applying settings has no relation with `addOnIndicatorAccuracyRadiusChangedListener / removeOnIndicatorAccuracyRadiusChangedListener`